staramr/blast/results/plasmidfinder/PlasmidfinderHitHSP.py

```diff
@@ -32,6 +32,10 @@ def __init__(self, file: str, blast_record: pd.Series) -> None:
 
         length = len(re_search)
 
+        if length == 2:
+            re_search.insert(1,'1')
```

Comment:
Instead of inserting `1` here (which becomes the gene_variant), could you insert `''` (e.g., empty string) here?

Also, could you add a command just saying what this is doing (e.g., `# Add empty string if gene variant is missing`).

Reply:
Yeah will fix that in my next commit